adaptor-config.properties.sample

```diff
@@ -54,6 +54,21 @@ documentum.src =
 # are pushed to GSA.
 # documentum.pushLocalGroupsOnly = false
 
+# Custom query to send modified documents. Incremental updates uses this query
+# to find the modified documents and send to GSA. {0} will be replaced by the
+# modified date and {1} will be replaced by the object ID. The select list
+# needs to include r_modify_date_str, r_object_id, i_chronicle_id
+# and object_name.
```

Comment:
Done. Added the description about the property values here for now. We can add the Wiki page and remove from here.